pom.xml

```diff
@@ -140,4 +162,31 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+      <profile>
+          <id>cloudwatch-logs-settings</id>
```

Comment:
Compare to https://github.com/jenkinsci/artifact-manager-s3-plugin/blob/2567c4c52d797921eefbf4298deaa3137b1b377a/pom.xml#L361